docs/getting_started/getting_started.rst

```diff
@@ -66,7 +62,7 @@ OpenHatch holds development meetings on IRC; our goal is to hold these meetings
 
 
 Read more documentation
-=======================
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
In general I'm +1 on using more section heading types.

One thing I dislike about rST is that it doesn't insist on consistency between documents, which can confuse different people who are editing the same collection of documents, if different docs within the same doc/* directory use different paradigms.

I almost feel like we need an rST style guide, to avoid this kind of internal inconsistency.